### ISSUE TITLE
Footer is not visible in mobile

### ISSUE THREAD

**Issue by carlosreig** [VLCTechHub/VLCTechHub-site#71]
The footer contains very valuable links (like the one that links to the Slack community) that are not displayed in the mobile version. Also, they are not available in the menu neither.

I guess the decision of hiding the footer is made in purpose, so I propose to add the links in the menu when the website is displayed in a mobile device.

**Comment by hell03610** [comment]
Yes, they are hidden right now on purpose... because we have a very basic menu. I'd like to rethink the menus and footers in general, to have an about page and other places to discover the valuable info more instead of in a footer (slack, browser extensions...) 

I think we need to get a designer in this conversation :D - already talking with some communities in valencia trying to spot one, but please let me know if you know anyone interested in helping so we can advance on this :)
